site/content/blog/making-the-most-out-of-clis.md

```diff
@@ -0,0 +1,93 @@
+Once the user confirms the changes, `kapp` waits for resources to reach their desired state ensuring that the changes are in effect as the user intended.
+
+### View the grouped resources and their statuses easily
+The `kapp inspect` command enables users to view resources that have been grouped into an "application". It lists the resources and surfaces information about whether or not the resources are in a healthy state. In case a resource is not in it's desired state, helpful information which helps the user work towards the cause of the failure is surfaced as well.
+
+## `kctrl`
+So what does our newest ally bring to the table?
+`kctrl` allows our users to interact with easily with `kapp-controllers` custom resources. It allows users to easily consume published Carvel packages and (will eventually help) interact with and debug [AppCRs](https://carvel.dev/kapp-controller/docs/latest/app-overview/) in a comprehensive manner.
+
+### Get up and running with Carvel packages easily
+Initially, installing a package would require the consumer to author a PackageInstall CR which would refer to a particular Package and a version, along with Secrets or ConfigMaps defining values consumed by the PackageInstall and an assigned service account it uses to make changes.
+`kctrl` allows users to easily install a package and conveniently supply details required to install it. Carvel packages can be configured by supplying certain values to them during installation. The CLI allows users to easily supply these values as a YAML file. This file is used to create a secret which is referenced by the PackageInstall created under the hood.
+This enables package consumers to be up and running, even if they are not ready to get their hands dirty with some YAML.
+
+### Have a reference while authoring values for packages
+As we discussed earlier, some packages accept configuration in the form of a YAML file defining certain values. The structure of such files for a package is defined using an OpenAPI schema while authoring these packages. `kctrl` let's user surface details about the values accepted along with short descriptions of the effect they have.
+
+### Re-configure and fix installations with ease
+`kctrl` also allows users to update the values supplied to a PackageInstall and bump versions for an installation easily as well. Of course, without the user having to author resources themselves.
+
+### What is coming up for `kctrl`?
+AppCRs are at the heart of Carvel's Packaging API. Different versions of a Package, are essentially different different workflows defined in an AppCR style so that they can be distributed - grouped as a PackageRepository if needed. And each PackageInstall resource decides which version of the AppCR should be running on a cluster.
```

Comment:
*nit*:
```suggestion
AppCRs are at the heart of Carvel's Packaging API. Different versions of a Package, are essentially different workflows defined in an AppCR style so that they can be distributed - grouped as a PackageRepository if needed. And each PackageInstall resource decides which version of the AppCR should be running on a cluster.
```
*suggestion*:
```suggestion
AppCRs are at the heart of Carvel's Packaging API. Different versions of a Package, are essentially different workflows defined in an AppCR style so that they can be distributed. And each PackageInstall resource decides which version of the AppCR should be running on a cluster. Multiple packages can be grouped together and distributed as a PackageRepository.
```

Reply:
These make sense, will do